test/e2e/common/volumes.go

```diff
@@ -394,6 +394,40 @@ var _ = framework.KubeDescribe("GCP Volumes", func() {
 	})
 
 	////////////////////////////////////////////////////////////////////////
+	// NFSv3
+	////////////////////////////////////////////////////////////////////////
+
+	framework.KubeDescribe("NFSv3", func() {
+		It("should be mountable for NFSv3", func() {
+			config := VolumeTestConfig{
+				namespace:   namespace.Name,
+				prefix:      "nfs",
+				serverImage: "gcr.io/google_containers/volume-nfs:0.8",
```

Comment:
In line 421, I put /exports in the mount path, so this is testing v3. Also in serverPorts, I also have to put 20048 in serverports for v3 (which is not required by v4)